api/services/Reddit.js

```diff
@@ -34,7 +35,7 @@ exports.refreshToken = async function(refreshToken) {
   }
 };
 
-var makeRequest = async function (refreshToken, requestType, url, data, rateLimitRemainingThreshold) {
+var makeRequest = async function (refreshToken, requestType, url, data, rateLimitRemainingThreshold, silentErrors) {
```

Comment:
Ah right. Can we call it "silenceErrors" rather than "silentErrors" as the latter implies that we are passing an error through itself.